aws/templates/setContext.ftl

```diff
@@ -253,6 +253,11 @@
             "Type" : ARRAY_OF_STRING_TYPE,
             "Mandatory" : true
         },
+        {
+            "Names" : "OWASP",
```

Comment:
Instead of hardcoding a security service maybe we should have security profiles which match to security standards. Then the rules defined in them can be seen easily in master data files and changed if requried

Reply:
Yep - plan  to change this in next PR. For now this attribute isn't checked so its harmless.